config/jobs/preflight/preflight-presubmits.yaml

```diff
@@ -0,0 +1,29 @@
+presubmits:
+  jetstack/preflight:
+
+  - name: preflight-unit-tests
+    cluster: gke
+    agent: kubernetes
+    decorate: true
+    decoration_config:
+      ssh_key_secrets:
+      - jetstack-pulling-bot
+    clone_uri: "git@github.com:jetstack/preflight.git"
```

Comment:
You can remove this + the line above (as it won't work anyway as that secret doesn't exist in the gke build cluster). The repo will be public anyway 😄